sdk/storage/storage-queue/test/utils/testutils.common.ts

```diff
@@ -41,3 +41,15 @@ export class ConsoleHttpPipelineLogger implements IHttpPipelineLogger {
     }
   }
 }
+
+export function getSASConnectionStringFromEnvironment(): string {
+  const connectionStringEnvVar = `STORAGE_SAS_CONNECTION_STRING`;
+  const env = isBrowser() ? (window as any).__env__ : process.env;
+  const connectionString = env[connectionStringEnvVar];
+
+  if (!connectionString) {
+    throw new Error(`${connectionStringEnvVar} environment variables not specified.`);
+  }
+
+  return connectionString;
+}
```

Comment:
Can we generate a SAS connection string dynamically using storage account & key in Node.js runtime?

Account Name + Account Key => SAS

So we can reuse current account_name and account_key environment variable, instead of importing a new REQUIRED environment variable.

SAS aims for a short term period access, and can be oudated when storage key is rotated. Also a SAS has a valid period, after that, SAS will be outdated. 

In browser runtime, we can reuse existing environment variables account_name with account_sas to generate the SAS connection string too.